geonode/base/templates/base/resourcebase_info_panel.html

```diff
@@ -35,11 +35,16 @@
     <dd>{{ resource.display_type }}</dd>
     {% endif %}
 
-    {% if resource.keyword_list %}
+    {% if resource.keywords %}
     <dt>{% trans "Keywords" %}</dt>
-    <dd>{% for keyword in resource.keyword_list %}
-        {{ keyword }}
-        {% endfor %}</dd>
+    <dd>
+      {% for keyword in resource.keywords.all %}
+        <a href="/search/?keywords__slug__in={{ keyword.slug }}">
```

Comment:
sure, there are more ones already in the same file. Do the merge and will
take care of them in the next days.

On Fri, Apr 17, 2015 at 4:51 PM, Simone Dalmasso <notifications@github.com>
wrote:

> In geonode/base/templates/base/resourcebase_info_panel.html
> <https://github.com/GeoNode/geonode/pull/2127#discussion_r28600563>:
>
> >      <dt>{% trans "Keywords" %}</dt>
> > -    <dd>{% for keyword in resource.keyword_list %}
> > -        {{ keyword }}
> > -        {% endfor %}</dd>
> > +    <dd>
> > +      {% for keyword in resource.keywords.all %}
> > +        <a href="/search/?keywords__slug__in={{ keyword.slug }}">
>
> Could you use the reverse url here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GeoNode/geonode/pull/2127/files#r28600563>.
>



-- 
Paolo Corti
Geospatial software developer
web: http://www.paolocorti.net
twitter: @capooti
skype: capooti